src/videouplinkbandwidthpolicy/DefaultSimulcastUplinkPolicy.ts

```diff
@@ -82,160 +62,84 @@ export default class DefaultSimulcastUplinkPolicy implements SimulcastUplinkPoli
     ]);
   }
 
-  updateConnectionMetric({ uplinkKbps = 0 }: ConnectionMetrics): void {
-    if (isNaN(uplinkKbps)) {
-      return;
-    }
-
-    // Check if startup period in order to ignore estimate when video first enabled.
-    // If only audio was active then the estimate will be very low
-    if (this.startTimeMs === 0) {
-      this.startTimeMs = Date.now();
-    }
-    if (Date.now() - this.startTimeMs < DefaultSimulcastUplinkPolicy.startupDurationMs) {
-      this.lastUplinkBandwidthKbps = DefaultSimulcastUplinkPolicy.defaultUplinkBandwidthKbps;
-    } else {
-      this.lastUplinkBandwidthKbps = uplinkKbps;
-    }
-    this.logger.debug(() => {
-      return `simulcast: uplink policy update metrics ${this.lastUplinkBandwidthKbps}`;
-    });
-
-    let holdTime = DefaultSimulcastUplinkPolicy.holdDownDurationMs;
-    if (this.currentActiveStreams === ActiveStreams.kLow) {
-      holdTime = DefaultSimulcastUplinkPolicy.holdDownDurationMs * 2;
-    } else if (
-      (this.currentActiveStreams === ActiveStreams.kMidAndLow &&
-        uplinkKbps <= this.midDisabledRate) ||
-      (this.currentActiveStreams === ActiveStreams.kHiAndLow && uplinkKbps <= this.hiDisabledRate)
-    ) {
-      holdTime = DefaultSimulcastUplinkPolicy.holdDownDurationMs / 2;
-    }
-    if (Date.now() < this.lastUpdatedMs + holdTime) {
-      return;
-    }
-
-    this.newQualityMap = this.calculateEncodingParameters(false);
-  }
+  updateConnectionMetric(_metrics: ConnectionMetrics): void {}
 
-  private calculateEncodingParameters(
-    numSendersChanged: boolean
-  ): Map<string, RTCRtpEncodingParameters> {
+  private calculateEncodingParameters(): Map<string, RTCRtpEncodingParameters> {
     // bitrates parameter min is not used for now
     const newBitrates: BitrateParameters[] = [
       new BitrateParameters(),
       new BitrateParameters(),
       new BitrateParameters(),
     ];
 
-    let hysteresisIncrease = 0,
-      hysteresisDecrease = 0;
-    if (this.currentActiveStreams === ActiveStreams.kHi) {
-      // Don't trigger redetermination based on rate if only one simulcast stream
-      hysteresisIncrease = this.lastUplinkBandwidthKbps + 1;
-      hysteresisDecrease = 0;
-    } else if (this.currentActiveStreams === ActiveStreams.kHiAndLow) {
-      hysteresisIncrease = this.enableFhdVideo ? 4000 : 2400;
-      hysteresisDecrease = this.hiDisabledRate;
-    } else if (this.currentActiveStreams === ActiveStreams.kMidAndLow) {
-      hysteresisIncrease = this.enableFhdVideo ? 1600 : 1000;
-      hysteresisDecrease = this.midDisabledRate;
+    if (this.shouldDisableSimulcast) {
+      // See comment above `shouldDisableSimulcast` for usage.
+      //
+      // The value of `newActiveStreams` is somewhat irrelevant since in one to one calls
```

Comment:
I think i mitigated this issue with some recent backend changes. I'll remove this mega comment another time though.